src/sentry/api/bases/user.py

```diff
@@ -43,8 +43,11 @@ def has_org_permission(self, request: Request, user):
         organization = organization_service.get_organization_by_id(
             id=organization_id, user_id=request.user.id
         )
+        if not organization:
+            return False
 
         self.determine_access(request, organization)
+        assert request.method is not None
```

Comment:
Best I can tell, this should never be possible anywhere except within tests

Reply:
Yeah real HTTP methods would have a method.